custom_components/foxess_modbus/entities/remote_control_description.py

```diff
@@ -53,7 +53,7 @@
                 max_soc=41010,
                 invbatpower=[31022],
                 battery_soc=[31024],
-                pwr_limit_bat_up=[44012],
```

Comment:
Mind just adding a comment with a link to the discussion where we discovered this? Or I'm happy to.

Reply:
Sorry should have added that 😊 

As discussed in this thread https://github.com/nathanmarlor/foxess_modbus/discussions/666#discussioncomment-10471917

Reply:
Ah sorry, I meant to the code. I'd do it myself, but you need to tick the little box allowing other people to make changes to the PR